static/app/components/forms/selectControl.tsx

```diff
@@ -81,6 +81,12 @@ export type ControlProps<OptionType = GeneralSelectValue> = Omit<
    * Show line dividers between options
    */
   showDividers?: boolean;
+  /**
+   * If false (default), checkmarks/checkboxes will be vertically centered
+   * wrt the first line of the label text. If true, they will be centered
+   * wrt the entire height of the option wrap.
+   */
+  verticallyCenterCheckWrap?: boolean;
```

Comment:
Useful for when we have leading items that are taller than the label line height.

When `false`:

<img width="279" alt="Screen Shot 2022-01-12 at 1 37 50 PM" src="https://user-images.githubusercontent.com/44172267/149225176-eb76bb2c-54a0-4090-8527-afe6a8d68d02.png">

When `true`:

<img width="279" alt="Screen Shot 2022-01-12 at 1 37 22 PM" src="https://user-images.githubusercontent.com/44172267/149225109-3e8595db-0a87-418c-a69d-b9070523145d.png">